packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -2375,7 +2386,7 @@ function updateDehydratedSuspenseComponent(
       retry = Schedule_tracing_wrap(retry);
     }
     registerSuspenseInstanceRetry(suspenseInstance, retry);
-    return null;
+    return reuseCurrentChild(workInProgress);
```

Comment:
nit: Seems odd to call this and have it return null for us. Let's just call this and return `null` on the next line?